apps/wallet/src/ui/app/pages/swap/QuoteAssets.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { coinsMap } from '_app/hooks/useDeepBook';
+import Overlay from '_components/overlay';
+import { useActiveAddress, useCoinsReFetchingConfig } from '_hooks';
+import { TokenRow } from '_pages/home/tokens/TokensDetails';
+import { useSuiClientQuery } from '@mysten/dapp-kit';
+import { useSearchParams } from 'react-router-dom';
+
+const recognizedCoins = Object.values(coinsMap);
```

Comment:
We should probably pass these as props even though it's a static list, just so that when we do make this dynamic it's less of an issue